tests/foreman/ui/test_contenthost.py

```diff
@@ -135,11 +172,27 @@ def test_positive_end_to_end(session, default_location, module_repos_collection_
 
     :CaseImportance: Critical
     """
+    property_name = setting_update.name
+    _distro = module_repos_collection_with_manifest.distro
+    host_rhel_version = None
+    if _distro.startswith('rhel'):
+        host_rhel_version = int(_distro[4:])
+    rhel_status = get_rhel_lifecycle_support(host_rhel_version if not None else 0)
+
     result = vm.run(f'yum -y install {FAKE_1_CUSTOM_PACKAGE}')
     assert result.status == 0
     startdate = datetime.utcnow().strftime('%m/%d/%Y')
     with session:
+        session.settings.update(f'name = {property_name}', 'Yes')
```

Comment:
Is this necessary? `new_hosts_page` should already be set to `True` by fixture `setting_update`, shouldn't it?